app/Http/Controllers/StatsticController.php

```diff
@@ -7,7 +7,7 @@
 
 class StatsticController extends Controller
 {
-    public function show(){
+    public function show() {
```

Comment:
__[phpmd]__
* The method show() has 107 lines of code. Current threshold is set to 100. Avoid really long methods.

<!-- Comment by [SideCI](https://sideci.com) -->